components/LandingPage/sections/Brands.tsx

```diff
@@ -0,0 +1,86 @@
+import Image, { StaticImageData } from 'next/image'
+import * as React from 'react'
+
+import ExtendImage from '../../../assets/Landing/brands/extend.png'
+import LampImage from '../../../assets/Landing/brands/lamp.png'
+import ChainImage from '../../../assets/Landing/brands/chain.png'
+import NetworkingImage from '../../../assets/Landing/brands/networking.png'
```

Comment:
in NextJS I believe it's best practice to store assets (like PNG / SVG) in the `public` directory.

Then, you can reference those assets with `/path-from-public.png`

https://nextjs.org/docs/app/building-your-application/optimizing/static-assets